content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md

```diff
@@ -132,10 +131,9 @@ In the bootstrap initialization process, the following occurs:
 9. Certificate is created for the kubelet
 -->
 1. kubelet 启动
-2. kubelet 看到自己**没有**对应的 `kubeconfig` 文件
+2. kubelet 看到自己 **没有** 对应的 `kubeconfig` 文件
```

Comment:
Not necessary but acceptable.

Ref: [current page](https://kubernetes.io/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#bootstrap-initialization)